distribution/linux/Ryujinx.sh

```diff
@@ -11,4 +11,9 @@ if [ -f "$SCRIPT_DIR/Ryujinx.Headless.SDL2" ]; then
     RYUJINX_BIN="Ryujinx.Headless.SDL2"
 fi
 
-env DOTNET_EnableAlternateStackCheck=1 "$SCRIPT_DIR/$RYUJINX_BIN" "$@"
+if command -v gamemoderun >/dev/null 2>&1
+then
+    env DOTNET_EnableAlternateStackCheck=1 gamemoderun "$SCRIPT_DIR/$RYUJINX_BIN" "$@"
+else
+    env DOTNET_EnableAlternateStackCheck=1 "$SCRIPT_DIR/$RYUJINX_BIN" "$@"
+fi
```

Comment:
How do you feel about something like this?

```suggestion
if command -v gamemoderun > /dev/null 2>&1; then
    COMMAND="$COMMAND gamemoderun"
fi

$COMMAND "$SCRIPT_DIR/$RYUJINX_BIN" "$@"
```

Additionally you'd add this line after line 4:
```sh
COMMAND="env DOTNET_EnableAlternateStackCheck=1"
```

(GitHub doesn't let me comment on unmodified lines, so I had to add that part to this comment as well.)

Reply:
Updated based on your suggestion, looks good to me.